src/icons/index.js

```diff
@@ -15,9 +15,11 @@ export * from './home';
 export * from './info';
 export * from './like';
 export * from './link';
+export * from './list';
```

Comment:
Can you make a list of the newly added icons? This will make it easier to update the icon document in the future. ☺️

Reply:
ok i will compare the current version with the documented one and post a diff

Reply:
new:

```
List: (props: IconProps) => <SvgIcon xml={list} {...props} />,
...
Member: (props: IconProps) => <SvgIcon xml={member} {...props} />,
...
Send: (props: IconProps) => <SvgIcon xml={send} {...props} />,
...
VolunteerCalendar: (props: IconProps) => <SvgIcon xml={volunteerCalendar} {...props} />
```

removed:

```
VolunteerConversationNew: (props: IconProps) => (
  <NamedIcon
    name={device.platform === 'ios' ? 'ios-mail-outline' : 'md-mail-outline'}
    {...props}
  />
),
VolunteerList: (props: IconProps) => (
  <NamedIcon
    name={device.platform === 'ios' ? 'ios-list-outline' : 'md-list-outline'}
    {...props}
  />
),
VolunteerLogout: (props: IconProps) => (
  <NamedIcon
    name={device.platform === 'ios' ? 'ios-lock-open-outline' : 'md-lock-open-outline'}
    {...props}
  />
),
VolunteerPersonal: (props: IconProps) => (
  <NamedIcon
    name={device.platform === 'ios' ? 'ios-person-outline' : 'md-person-outline'}
    {...props}
  />
)
```

Reply:
Thank you ☺️